chorus_lib/src/transport/http.rs

```diff
@@ -80,23 +98,27 @@ impl HttpTransport {
         let agent = AgentBuilder::new().build();
 
         Self {
-            config,
+            config: info.clone(),
```

Comment:
You're taking the ownership of `http_config` so you probably don't need to clone. It seems that `info` is not used anywhere else so you might just want to delete `info` and put `http_config.info` here.

```suggestion
            config: http_config.info,
```